bentoml/_internal/server/__init__.py

```diff
@@ -159,6 +163,7 @@ def serve_production(
             numprocesses=app_workers or 1,
             stop_children=True,
             use_sockets=True,
+            working_dir=working_dir,
```

Comment:
@bojiang we need this change per https://github.com/bentoml/BentoML/pull/2064#discussion_r759091488